static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,63 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const GITHUB_CLOUD = ["github.com", "www.github.com"];
+
+/**
+ * Method that checks the validity of the passed URL
+ *
+ * @returns {boolean}
+ */
+const checkValidUrl = inputURL => {
```

Comment:
We need to add typing here. What are we passing in and what are we returning?

Reply:
Defined the types in the annotations above...